packages/plugin-packer/src/create-contents-zip.ts

```diff
@@ -0,0 +1,32 @@
+import path from "path";
+import { ZipFile } from "yazl";
+import streamBuffers from "stream-buffers";
+import { sourceList } from "./sourcelist";
+import _debug from "debug";
+
+const debug = _debug("create-contents-zip");
+
+/**
+ * Create a zipped contents
+ */
+export function createContentsZip(
+  pluginDir: string,
+  manifest: any
+): Promise<false | Buffer> {
+  return new Promise((res, rej) => {
+    const output = new streamBuffers.WritableStreamBuffer();
+    const zipFile = new ZipFile();
+    let size: any = null;
+    output.on("finish", () => {
+      debug(`plugin.zip: ${size} bytes`);
+      res(output.getContents());
+    });
+    zipFile.outputStream.pipe(output);
+    sourceList(manifest).forEach((src) => {
+      zipFile.addFile(path.join(pluginDir, src), src);
+    });
+    zipFile.end(undefined, ((finalSize: any) => {
```

Comment:
The finalSize is a number or null.

https://github.com/thejoshwolfe/yazl/blob/36b2a6a31a32637dca19b34e7b3295378e78a644/index.js#L177
https://github.com/thejoshwolfe/yazl/blob/36b2a6a31a32637dca19b34e7b3295378e78a644/index.js#L216
```suggestion
    zipFile.end(undefined, ((finalSize: number) => {
```

Reply:
Thanks! 16b4530